src/constants.js

```diff
@@ -370,3 +370,9 @@ export const KEYS = {
 
 /* ----------------- Other ----------------------- */
 export const ONE_HOUR_MS = 3600000; // 60 * 60 * 1000
+
+/* ----------------- Fuzzy Search/Suggested Collabs ----------------------- */
```

Comment:
This constants file is intended for global use; these values seem very specific to USM and should probably live in its feature directory.

Reply:
Yeah I moved it here because it used to be in the USM folder but needed to move out to be referenced by utils as well. Perhaps it should move to utils? I was also a little unsure on if I wanted to move thee values out to the consumers instead of them living in the util.